601-spring-data-primitive-types/src/test/java/io/quarkus/qe/spring/data/BookResourceTest.java

```diff
@@ -48,11 +54,43 @@ void testCustomFindPublicationIsbnObjectLong() {
     // QUARKUS-525
```

Comment:
Please, add "TODO" and the fully qualified url of the issue, this way is much easier to search.